doc/man3/BIO_s_dgram.pod

```diff
@@ -0,0 +1,235 @@
+=pod
+
+=head1 NAME
+
+BIO_s_datagram - Network BIO with datagram semantics
+
+=head1 SYNOPSIS
+
+ #include <openssl/bio.h>
+
+ BIO_METHOD *BIO_s_datagram(void);
+ BIO *BIO_new_dgram(int fd, int close_flag);
+
+ int BIO_ctrl_dgram_connect(BIO *bio, const BIO_ADDR *peer);
+ int BIO_ctrl_set_connected(BIO *bio, const BIO_ADDR *peer);
+ int BIO_dgram_recv_timedout(BIO *bio);
+ int BIO_dgram_send_timedout(BIO *bio);
+ int BIO_dgram_get_peer(BIO *bio, BIO_ADDR *peer);
+ int BIO_dgram_set_peer(BIO *bio, const BIO_ADDR *peer);
+ int BIO_dgram_get_mtu_overhead(BIO *bio);
+
+=head1 DESCRIPTION
```

Comment:
This section does not follow the usual pattern that each function listed in the synopsis, re-appears here, in the same order, with a paragraph about what it does. Is this on purpose?

Reply:
I guess I'm not sure that's the best way to provide exposition in this circumstance.

Reply:
Yeah, could well be that in this case a more holistic presentation is best.
The only drawback is that if a reader just wants to read what a particular function does, (s)he will have some difficulties finding the right place to look.
Yet maybe this is hard to avoid - I had this problem also when writing other rather complex documentation, e.g., on the new HTTP client API.